configs/SPRACINGH7NANO/config.h

```diff
@@ -72,16 +72,19 @@
 #define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
 #endif
 
+#define USE_SPI_DEVICE_2
```

Comment:
This is already defined in target.h

Reply:
`target.h` ???  what file /exactly/ are you referring to?  (full path, url, line number) please.  I don't have any target.h files, only `<target>.config` and `<target>\config.h`.

Reply:
@hydra look here: https://github.com/betaflight/betaflight/blob/master/src/main/target/STM32H750/target.h

Reply:
yes I'm aware of the base target, but they are gated by `#if !defined(USE_SPI)`, so you need them, as `#define USE_SPI` is defined.

This is by design, so that you don't include code for SPI ports that are not used on the target which just wastes RAM and FLASH.